js/apps/system/_admin/aardvark/APP/frontend/js/views/notificationView.js

```diff
@@ -15,7 +15,10 @@
       this.collection.bind('reset', this.renderNotifications.bind(this));
 
       window.setTimeout(function () {
-        if (frontendConfig.authenticationEnabled === false && frontendConfig.isCluster === false && arangoHelper.showAuthDialog() === true) {
+        if (frontendConfig.authenticationEnabled === false &&
```

Comment:
Has this changed? Or just pushed to another line?

Reply:
Just pushed to another line, for better readability

Reply:
I can revert this